Documentation/ApiOverview/Authentication/CSRFlikeRequestTokenHandling.rst

```diff
@@ -0,0 +1,163 @@
+..  include:: /Includes.rst.txt
+..  index:: Authentication, Request token
+..  _authentication-request-token:
+
+================================
+CSRF-like request token handling
+================================
+
+..  versionadded:: 12.0
+
+A CSRF-like request token handling is available to mitigate potential cross-site
+requests on actions with side effects. This approach does not require an
+existing server-side user session, but uses a nonce (number used once) as a
+"pre-session". The main scope is to ensure a user actually has visited a page,
+before submitting data to the webserver.
+
+This token can only be used for HTTP methods `POST`, `PUT` or `PATCH`, but
+for instance not for `GET` request.
+
+The :php:`\TYPO3\CMS\Core\Middleware\RequestTokenMiddleware` resolves
```

Comment:
link this to github?

Reply:
Does it has any value? It is no API, it just does the work?